src/pages/students/forms/studentDataForm/StudentData.tsx

```diff
@@ -5,26 +5,37 @@ import { MainContainer,
         Description, 
         InputDate, 
         Button, 
-        Title, 
         ButtonsContainer, 
-        TitleContainer, 
         Label} 
-        from "../styles/StudentData.styles";
-import { StudentDataScheme } from "../schemes/StudentData.scheme";
+        from "./StudentData.styles";
+import { StudentDataScheme } from "./StudentData.scheme";
 import { useForm } from "react-hook-form";
 import * as yup from "yup";
-import { ErrorMessage } from "../../../components/header/ErrorMessage";
+import { ErrorMessage} from "../../../../components/error_message/ErrorMessage";
 import { useNavigate } from "react-router";
-import { useDispatch, useSelector } from "react-redux";
-import { resetStudentData, setStudentData } from "../StudentRegistrationFormSlice";
-import type { RootState } from "../../../app/store/store";
+import type { StudentDataProps } from "../../create-student/CreateStudent";
 
+interface FormProp {
+  createStudentCall: (data: StudentDataProps) => void;
+  studentData: StudentDataProps | undefined;
+}
 
-function StudentData() {
+function StudentData({
+  createStudentCall,
+  studentData
+}: FormProp) {
```

Comment:
Y en este formulario lo mismo


```suggestion
function StudentData({
  onNext,
  onBack,
  data
}: FormProp<StudentDataProps>) {
```